recipe/meta.yaml

```
@@ -32,11 +33,21 @@ requirements:
     - requests >=2.8.1
 
 test:
+  source_files:
+    - tests
+    - sample-data
   imports:
     - wfdb
     - wfdb.plot
     - wfdb.processing
     - wfdb.io
+  requires:
+    - pip <24.2
+    - pytest
```

```suggestion
    - pytest
    - pytest-xdist >=2.5.0
```
pytest-xdist is also nice to have during tests to speed up them